Qt/mainwindow.h

```diff
@@ -90,27 +90,37 @@ private slots:
 	void takeScreen() { g_TakeScreenshot = true; }
 	void consoleAct();
 
-	// Options
-	// Core
-	void vertexDynarecAct() { g_Config.bVertexDecoderJit = !g_Config.bVertexDecoderJit; }
-	void fastmemAct() { g_Config.bFastMemory = !g_Config.bFastMemory; }
-	void ignoreIllegalAct() { g_Config.bIgnoreBadMemAccess = !g_Config.bIgnoreBadMemAccess; }
-
-	// Video
-	void anisotropicGroup_triggered(QAction *action) { g_Config.iAnisotropyLevel = action->data().toInt(); }
+	// Game settings
+	void keepOnTopAct() { g_Config.bTopMost = !g_Config.bTopMost; }
```

Comment:
I doubt this one works on Qt without implementing, but probably just need a `setWindowFlags(Qt::WindowStaysOnTopHint);` somewhere.  It needs to be in the menu switch and also on startup.

-[Unknown]

Reply:
Removed for the future.